statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1985,139 +2596,220 @@ def plot_acorr(self, nlags=10, resid=True, linewidth=8):
         return fig
 
     def test_normality(self, signif=0.05):
         """
-        Test assumption of normal-distributed errors using Jarque-Bera-style
-        omnibus Chi^2 test.
+        Test assumption of normality in resids with Jarque-Bera-style test
 
         Parameters
         ----------
         signif : float
-            Test significance level.
+            Test significance threshold
 
         Returns
         -------
         result : NormalityTestResults
+            Result instance containing the test statistic
 
         Notes
         -----
         H0 (null) : data are generated by a Gaussian-distributed process
+
+        References
+        ----------
+        .. [*] Lütkepohl, H., 2005. New introduction to multiple time series
+           analysis. Springer Science & Business Media.
+
+        .. [*] Kilian, L. & Demiroglu, U. (2000). "Residual-Based Tests for
+           Normality in Autoregressions: Asymptotic Theory and Simulation
+           Evidence." Journal of Business & Economic Statistics
         """
         return test_normality(self, signif=signif)
 
     @cache_readonly
     def detomega(self):
+        """
+        detomega is deprecated. Use `det_cov_resid`
+        """
+        import warnings
+        warnings.warn("detomega is deprecated and will be removed in 0.11.0. "
+                      "Use det_cov_resid.", DeprecationWarning)
+        return self.det_cov_resid
+
+    @cache_readonly
+    def det_cov_resid(self):
         r"""
-        Return determinant of white noise covariance with degrees of freedom
-        correction:
+        Returns determinant of the cov_resid with degrees of freedom correction
 
         .. math::
 
             \hat \Omega = \frac{T}{T - Kp - 1} \hat \Omega_{\mathrm{MLE}}
+
+        where :math:`\Omega` is the covariance of the residuals
         """
-        return scipy.linalg.det(self.sigma_u)
+        return scipy.linalg.det(self.cov_resid)
 
     @cache_readonly
     def info_criteria(self):
-        "information criteria for lagorder selection"
+        """
+        Information criteria for order selection
+
+        Returns
+        -------
+        ic : dict-like
+            Container with all model information criteria
+        """
         nobs = self.nobs
         neqs = self.neqs
         lag_order = self.k_ar
-        free_params = lag_order * neqs ** 2 + neqs * self.k_exog
+        free_params = lag_order * neqs**2 + neqs * self.k_exog
 
-        ld = logdet_symm(self.sigma_u_mle)
+        ld = logdet_symm(self.cov_resid_mle)
 
-        # See Lutkepohl pp. 146-150
+        # See Lütkepohl pp. 146-150
 
         aic = ld + (2. / nobs) * free_params
         bic = ld + (np.log(nobs) / nobs) * free_params
         hqic = ld + (2. * np.log(np.log(nobs)) / nobs) * free_params
-        fpe = ((nobs + self.df_model) / self.df_resid) ** neqs * np.exp(ld)
+        fpe = ((nobs + self.df_model) / self.df_resid)**neqs * np.exp(ld)
+        res = Bunch()
```

Comment:
-1

Reply:
I don't like the name 'Bunch'. Use a more complicated version that handles more edge cases correctly https://github.com/bashtage/linearmodels/blob/master/linearmodels/utility.py#L21 called `AttrDict` which describes its purpose.